pyacq/core/stream.py

```diff
@@ -134,8 +134,6 @@ def _compress_blosclz4(self, index, data):
         return index, data
     
     def close(self):
-        if self.params['protocol'] == 'tcp':
-            self.socket.unbind(self.url)
```

Comment:
I had to remove these to avoid a zmq bug:
https://github.com/zeromq/libzmq/issues/949

Any reason you think it is needed? I presume calling `socket.close()` will unbind it as well.